docs/blog/posts/multi-stage-delivery-using-gitops.md

```diff
@@ -0,0 +1,413 @@
+The Helm charts can be found in our [GitOps repository](https://github.com/bacherfl/keptn-analysis-demo)
+in the following sub folders:
+
+- `simple-app/chart-dev`: Contains the Helm chart for the application in the `dev` stage
+- `simple-app/chart-prod`: Contains the Helm chart for the application in the `prod` stage
+
+The ArgoCD applications are created by applying the following manifest:
+
+```yaml title="argo-apps.yaml"
+{% include "./multi-stage-delivery-using-gitops/argo-apps.yaml" %}
+```
+
+The manifest above contains the definition for the two ArgoCD applications
+each of which points to one of the helm charts mentioned earlier.
+In addition to that, the `$ARGOCD_APP_REVISION` environment variable
+is used to get access to the git commit ID that triggered
+a new deployment of our applications.
+This ID is passed through to the Helm chart and is used by
+Keptn to include this as metadata for a `KeptnApp` deployment.
+
+After applying the file, using `kubectl apply -f argo-apps.yaml`,
+ArgoCD begins to synchronize the state of the applications,
+meaning that the Helm charts for the applications are applied to the
+cluster.
+While this is happening, let's have a closer look at the actual
+content of the Helm charts.
+
+### What's in the Helm chart for dev stage
+
+Both charts contain two `Deployments/Services`
+(`simple-go-service` and `simple-go-backend`), representing
+the two `KeptnWorkloads` that are part of our `KeptnApp`.
+Let's take the `simple-go-service` `Deployment` as an example
+to see how we prepared it to be managed by Keptn:
+
+```yaml
+{% include "./multi-stage-delivery-using-gitops/deployment.yaml" %}
+```
+
+#### Labels
+
+To correctly associate the `Deployment` resource with the `KeptnWorkload` resource,
+the following labels are set:
+
+- `app.kubernetes.io/name`: The name of the `KeptnWorkload` that should be associated with the `Deployment`.
+- `app.kubernetes.io/part-of`: The name of the `KeptnApp` containing the two workloads.
+- `app.kubernetes.io/version`: The version for the related `KeptnWorkload`.
+
+#### Pre and post-deployment tasks
+
+In addition to the labels which define the `KeptnWorkload`, we also use
+the `keptn.sh/post-deployment-tasks` to define a post-deployment task for the
+workload.
+The task defined here (`wait-for-monitoring`) ensures the Prometheus
+target for the workload is available, before proceeding with
+the execution of the load tests of the overall application.
+
+#### KeptnAppContext
+
+The `KeptnAppContext`  provides two important capabilities for multi-stage delivery:
+
+- Define tasks and evaluations that run before or after the application deployment
+- Add metadata and links to traces for a specific application.
+This enables you to enrich your traces
+with additional information that you can use
+to understand and analyze the performance of your applications
+which looks as follows:
+
+```yaml
+{% include "./multi-stage-delivery-using-gitops/keptnappcontext.yaml" %}
+```
```

Comment:
```suggestion
  ```yaml
  {% include "./multi-stage-delivery-using- 
  gitops/keptnappcontext.yaml" %}
  ```
```
I think this will get the yaml code to be nested under the bullet point but check the output -- these indentations are really fickle!